src/Admin/AbstractAdmin.php

```diff
@@ -1654,7 +1681,7 @@ public function getFilterFieldDescriptions()
         return $this->filterFieldDescriptions;
     }
 
-    public function addChild(AdminInterface $child)
+    public function addChild(AdminInterface $child, $field = null)
```

Comment:
This is a BC break, wdyt on using `func_get_args()`? @greg0ire

Reply:
ok, but I was thinking, should this PR be in `master` branch? Because this also adds `AbstractAdmin::addParentAssociationMapping` which is not in the `AdminInterface` and maybe we should find a cleaner way to do this.

Reply:
It is recommended to do in a BC way on stable if it is doable

Reply:
You create a new interface just for this and have the admin class implement it.

Reply:
Maybe add all related method of the parent things?

Reply:
Why is this a BC break if the field is default null? It will work as before if you don't pass that field